index.html

```diff
@@ -26,50 +26,275 @@
+                  ><img src="./images/github.svg" alt="" class="icon-img"
+                /></a>
+              </li>
+              <li>
+                <a href="https://github.com/blessing-michael"
+                  ><img src="./images/Vector (3).svg" alt="" class="icon-img"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </article>
+      </section>
+      <section class="meee">
+      <section class="section-two">
+        <article class="article-1">
+          <img src="./images/work-1.png" alt="" class="article-img" />
+          <div class="info-div1">
+            <h2 class="title">tonic</h2>
+            <div class="info">
+              <h4 class="canopy">canopy</h4>
+              <p class="profession">
+                <span class="prof-span">
+                  <img src="./images/Counter.png" alt="" /></span
+                >back end dev
+              </p>
+              <p class="profession">
+                <span class="prof-span">
+                  <img src="./images/Counter.png" alt="" /></span
+                >2015
+              </p>
+            </div>
+            <div class="info-2">
+              <p class="info-p">
+                A daily selection of privately <br />
+                personalized reads; no accounts or sign-ups required.
+              </p>
+            </div>
+
+            <div class="ul">
+              <ul class="tech-stack">
+                <li class="tech-name">HTML</li>
+                <li class="tech-name">CSS</li>
+                <li class="tech-name">Javascript</li>
+              </ul>
+            </div>
+
+            <button class="project-btn">see project</button>
+          </div>
+        </article>
+      </section >
+      <section class="section-two two">
+        <article class="article-1">
+          <img src="./images/work-2.png" alt="" class="article-img" />
+          <div class="info-div1">
+            <h2 class="title">Multi-Post Stories</h2>
+            <div class="info">
+              <h4 class="canopy">canopy</h4>
+              <p class="profession">
+                <span class="prof-span">
+                  <img src="./images/Counter.png" alt="" /></span
+                >back end dev
+              </p>
+              <p class="profession">
+                <span class="prof-span">
+                  <img src="./images/Counter.png" alt="" /></span
+                >2015
+              </p>
+            </div>
+            <div class="info-2">
+              <p class="info-p">
+                A daily selection of privately <br />
+                personalized reads; no accounts or sign-ups required.
+              </p>
+            </div>
+
+            <div class="ul">
+              <ul class="tech-stack">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+
+            <button class="project-btn">see project</button>
```

Comment:
- [ ] Please 🙏🏽 use capital letters for a better UI.

| Figma design | Your design |
|---|---|
| ![image](https://github.com/blessing-michael/portfolio/assets/58642949/875dea5c-c170-43c6-84cb-fbed6eff1823) | ![image](https://github.com/blessing-michael/portfolio/assets/58642949/9d7effb6-d9ec-40cb-b2a0-02e4d6086b93) |